DAL/SightseeingDBContext.cs

```diff
@@ -0,0 +1,229 @@
+﻿using DAL.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace DAL;
+
+public partial class SightseeingdbContext : DbContext
+{
+    public SightseeingdbContext()
+    {
+    }
+
+    public SightseeingdbContext(DbContextOptions<SightseeingdbContext> options)
+        : base(options)
+    {
+    }
+
+    public virtual DbSet<Guide> Guides { get; set; }
+
+    public virtual DbSet<Review> Reviews { get; set; }
+
+    public virtual DbSet<Role> Roles { get; set; }
+
+    public virtual DbSet<Sight> Sights { get; set; }
+
+    public virtual DbSet<Ticket> Tickets { get; set; }
+
+    public virtual DbSet<Tour> Tours { get; set; }
+
+    public virtual DbSet<User> Users { get; set; }
+
+    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder) 
+        => optionsBuilder.UseMySQL("Name=ConnectionStrings:DefaultConnection");
+
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
```

Comment:
It is also bad practice. You should have separate  configuration files as i show you

Reply:
You can apply them using one line of code